doc: zbus: swap boxes to match bullet points

andreaskurz opened the pull request:
By swapping the boxes with observer types in the graphics (thanks Rodrigo) it now matches the description of the bullet points. Before they were not in the order of the bullet points which confused during reading.

rodrigopex reviewed (changes requested):
Please use the adjusted image (above) instead of changing the text. Thank you for your contribution.

andreaskurz committed:
doc: zbus: swap boxes to match bullet points

By swapping the boxes with observer types in the graphics (thanks
Rodrigo) it now matches the description of the bullet points. Before
they were not in the order of the bullet points which confused during
reading.

Signed-off-by: Andreas Kurz <info@akurz.net>

rodrigopex reviewed (approved):
Thanks for contributing.
